input/docs/controls/ColorPicker.md

```diff
@@ -0,0 +1,492 @@
+Title: ColorPicker
+Description: The documentation for the ColorPicker, ColorCanvas, ColorPallette and ColorEyeDropper
+---
+**On this page**
+<!-- Start Document Outline -->
+
+* [Introduction](#introduction)
+* [ColorNamesDictionary and ColorHelper](#colornamesdictionary-and-colorhelper)
+	* [Looking up a colors name](#looking-up-a-colors-name)
+	* [Getting a color from a given name](#getting-a-color-from-a-given-name)
+	* [How to provide custom color names](#how-to-provide-custom-color-names)
+	* [Provide translations for a language of your choice](#provide-translations-for-a-language-of-your-choice)
+* [ColorCanvas](#colorcanvas)
+	* [The user interface](#the-user-interface)
+	* [Properties](#properties)
+	* [Events](#events)
+	* [DynamicResources](#dynamicresources)
+	* [Example](#example)
+* [ColorPalette](#colorpalette)
+	* [The user interface](#the-user-interface-1)
+	* [Properties](#properties-1)
+	* [DynamicResources](#dynamicresources-1)
+	* [Example](#example-1)
+	* [Build in color palettes](#build-in-color-palettes)
+* [ColorEyeDropper](#coloreyedropper)
+	* [The user interface](#the-user-interface-2)
+	* [Properties](#properties-2)
+	* [Events](#events-1)
+	* [DynamicResources](#dynamicresources-2)
+	* [Example](#example-2)
+* [ColorPicker](#colorpicker)
+	* [The user interface](#the-user-interface-3)
+	* [Events](#events-2)
+	* [DynamicResources](#dynamicresources-3)
+	* [Example](#example-3)
+
+<!-- End Document Outline -->
+---
+
+# Introduction
+
+The `ColorPicker`-Controls can be used to select any `System.Windows.Media.Color` (in the following lines called `Color`). There are four different controls available to select the a `Color`:
```

Comment:
typos:
- in this document simply called `Color`
- available to select a `Color`: